integration-tests/cstor-pool/cstor_integration_test.go

```diff
@@ -18,13 +18,10 @@ package cstorpoolit
 
 import (
 	"testing"
-
 	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/openebs/CITF"
```

Comment:
@aswathkk 
Why did we remove this import?
I guess we are using it.

Reply:
It is not used anywhere in the code. So VSCode removed it I guess.